cmd/podman/run_test.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"testing"
+
+	ociv1 "github.com/opencontainers/image-spec/specs-go/v1"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/stretchr/testify/assert"
+	"github.com/urfave/cli"
+)
+
+var (
+	cmd         = []string{"podman", "test", "alpine"}
+	CLI         *cli.Context
```

Comment:
I don't like using a global here... Do Go tests ever run in parallel? Can you make testCommand with a function so you get a differend cli.Context every time?